configure.ac

```diff
@@ -704,3 +704,10 @@ AC_CONFIG_FILES([
 fi
 
 AC_OUTPUT
+
+AC_MSG_WARN([
```

Comment:
When we release `8.0` (this pr is for `8.0a`), we could change our approach to something more pedantic: 
```
AC_ARG_ENABLE([enable-deprecated-autotools],
  [[enable-deprecated-autotools=no])

if test "x$enable-deprecated-autotools" != "xno"; then
  AC_MSG_ERROR([
    8.0 is the last release to support autotools. We highly recommend shifting to CMake as of now, please have a look at README.md.
    To continue using autotools, configure with --enable-deprecated-autotools.
  ])
fi
```
Users will have to pass in extra flag, and they will be most definitely be aware of the need to move to CMake (as compared to just a configure warning). @nrnhines do you think this is something worth including for this PR (8.0.a)?

Reply:
Discussed this offline with Alex : adding extra option like `--enable-deprecated-autotools` could be done once we deprecate auto tools. For now, we will print message at configure step.